src/Microsoft.DotNet.Wpf/src/PresentationCore/MS/internal/FontCache/FontSource.cs

```diff
@@ -381,6 +381,26 @@ private Stream GetCompositeFontResourceStream()
 
         #region Private Classes
 
+        /// <summary>
+        /// Calls <see cref="UnmanagedMemoryStream.Dispose"/> from the destructor.
+        /// </summary>
+        /// <remarks>TODO: Remove this once FontFileStream is migrated from C++/CLI.</remarks>
```

Comment:
Don't the XML comments get compiled into public docs? If so, perhaps simple comment would be better.

Reply:
Ugh, no idea to be honest as the docs that are shown are always different than the ones in the project, so I do not think so.

If that's the case however, changing it to a plain comment is in order indeed.